app/routes/automated-tools.js

```diff
@@ -72,8 +72,42 @@ export default class AutomatedToolsRoute extends Route {
         category: 'Testing Engine',
         description: 'A accessibility testing engine for JavaScript.',
         link: 'https://github.com/dequelabs/axe-core'
-      },
-      
+      }, {
+        title: 'alfa',
+        category: 'Testing Engine',
+        description: 'Suite of open and standards-based tools for performing reliable accessibility conformance testing at scale.',
+        link: 'https://github.com/Siteimprove/alfa'
+      }, {
+        title: 'AccessLint',
+        category: 'Code Linter (Static)',
+        description: 'AccessLint brings automated web accessibility testing into your development workflow. When a pull request is opened, AccessLint reviews the changes and comments with any new accessibility issues, giving you quick, timely, and targeted feedback, before code goes live.',
+        link: 'https://github.com/marketplace/accesslint'
+      }, {
+        title: 'equal-access',
+        category: 'Browser Extension, Testing Engine, Testing Library',
```

Comment:
This repo seems to be a monorepo containing all 3 of these things, so I wasn't sure how to reflect that in the category (and just left them all in instead, separated by commas)

Reply:
I need to refactor the way tags are done if we want to make them arrays. How about separating them according to the instructions in their own README for the time being? In the description you could add "part of the IBM equal-access suite" or something similar. WDYT?

Reply:
Yup that's a clever idea! I went ahead and split it into 3 with that trailing text being the same for all of them